ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -30,23 +32,21 @@
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
```

Comment:
Refactored the above code block so that the test logic is effective. Ideally transferToWardService test can be effective when interacting `PatientVisitDashboard` page **not** `ClinicianFacingPatientDashboard page`. The refactored `codeblock` creates `testvisit.uuid, visistType` and creating `SessionLocation `(`getLocationUuid(homePage`), But creating session location can only be effective when a test class extends `LocationSensitiveApplicationTestBase` so this block is not used by the test at all . The other two methods implementation have been catered within the test logic method `LN 40-43` and thus enabling the test to interact with `PatientVisitDashboard` page which holds functionalities that the test implementation need for it to be effective.